src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div
+    :key="item.key"
+    class="k-multiselect-item"
+    :data-testid="`k-multiselect-item-${item.value}`"
+    @click="handleClick"
+  >
+    <div
+      role="option"
+      class="d-block"
+    >
+      <button
+        :value="item.value"
+        :class="{ disabled, selected: item.selected }"
+      >
+        <span class="k-multiselect-item-label mr-2">
+          <slot name="content">{{ item.label }}</slot>
+        </span>
+        <span class="k-multiselect-selected-icon-container">
+          <KIcon
+            v-if="item.selected"
+            icon="check"
+            color="var(--blue-200)"
+            class="selected-item-icon"
+          />
+        </span>
+      </button>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import KIcon from '@/components/KIcon/KIcon.vue'
+
+export default defineComponent({
+  name: 'MultiselectItem',
+  components: { KIcon },
+  props: {
+    item: {
+      type: Object,
```

Comment:
suggestion: Can you type this item as `MultiselectItem` via `PropType`?

If yes, you're going to have to move the `MultiselectItem` type interface to a separate file that then gets imported into `KMultiselect.vue` and `KMultiselectItem.vue` to avoid a circular dependency.